fakestorage/upload_test.go

```diff
@@ -219,3 +219,55 @@ func TestServerInvalidUploadType(t *testing.T) {
 		t.Errorf("wrong status returned\nwant %d\ngot  %d", expectedStatus, resp.StatusCode)
 	}
 }
+
+func TestParseContentRange(t *testing.T) {
+	var goodHeaderTests = []struct {
```

Comment:
Minor, but can you mark this test (and its sub-tests) as parallel with ``t.Parallel()``?